### DISCUSSION TITLE
A detail row element is created even though the row collapses

### DISCUSSION THREAD

**Discussion by rdmzcn** [KevinVandy/material-react-table#1061] [Q&A] [answered]
I made a table with infinite scroll and detail row. Even though all the rows are collapsed, there are empty detail row fields under all of them. When I open the row, the normal detail area appears.

![1](https://github.com/KevinVandy/material-react-table/assets/9269722/8fc92ad5-f3da-4a6f-ba88-2d45b2a00423)

![2](https://github.com/KevinVandy/material-react-table/assets/9269722/328ac0cb-9dc2-4b63-a74c-0e2be2ae5048)

![3](https://github.com/KevinVandy/material-react-table/assets/9269722/1cf0214f-3429-4658-8e05-e4d0883a585d)

**Comment by KevinVandy** [comment]
This is intentional to avoid striped CSS bugs, and row virtualization bugs

**Reply by rdmzcn** [reply]
Thank you for your reply. I read what you said in the documentation. But in the examples there is no space like in my case.

**Reply by rdmzcn** [reply]
In the examples, when row collapses, padding-top and padding-bottom become 0. When expanded, it becomes 16-16. In my project, both row collapse and expand padding-top and padding-bottom are 14. This is the reason, but I could not find the solution.

**Reply by KevinVandy** [reply, marked as answer]
If you're getting weird padding, that's not right

**Reply by rdmzcn** [reply]
I am getting weird padding. I think it's due to my theme settings. 0.875rem padding crushed the padding of the table. I need to write correct paddings in the sx settings, but I couldn't find exactly what to write.

**Comment by rdmzcn** [comment]
I deleted the table paddings in my own css file and it was fixed. thanks @KevinVandy
